databases/business-sync/fragment-shoebox-item.js

```diff
@@ -30,6 +30,15 @@
       immutable: true,
       mustNotBeEmpty: true
     },
+    state: {
+      // The shoebox item's state within the shoebox processing pipeline. This state can be null for old items but will
```

Comment:
This currently won't allow backward compatible updates.  I'm assuming you intend on pushing this responsibility to the client (ie, explicitly handle adding a state with the appropriate value when none is found).  Alternatively, there is a validation property [`skipValidationWhenValueUnchanged`](https://github.com/Kashoo/synctos#universal-validation-constraints) that would allow backward compatibility saves, but perhaps we don't actually want to allow that?

Reply:
This setup allows null values for state. I added a unit test to make sure that that is the case, so it should be good. (we chatted about this offline)

Reply:
Aha, I misunderstood the default constraints for the `enum` type - this does indeed cover the empty case